spec/acceptance_checking_spec.rb

```diff
@@ -87,6 +107,13 @@ def license_plate_pattern
         expect { instance[:license_plate] = "invalid" }.to raise_error(exception, message, &further_expectations)
       end
 
+      it 'should give the user the location of the proc determining what it accepts on InvalidValueError' do
+        exception = SmartProperties::InvalidValueError
+        message = /Values passing lambda defined in \/Users\/mallen\/dev\/smart_properties\/spec\/acceptance_checking_spec\.rb at line 85/
```

Comment:
Haha, yes indeed of course it would. How silly of me. I'll just get it to match against the `/spec\/acceptance_checking_spec\.rb at line 85` part.